amethyst_ui/src/prefab.rs

```diff
@@ -359,6 +356,7 @@ where
 /// - `TF`: `Format` used for loading `Texture`s
 /// - `AF`: `Format` used for loading sounds
 /// - `FF`: `Format` used for loading fonts
+/// - `WID`: Type used for Widget IDs
```

Comment:
Why not just W? ;)

Reply:
Was trying to keep it consistent with the others, I have no idea why they are different from all others either tbh =P I'll change them all to be in line with the rest of the file.